content/blog/2022/11/deprecation-policy.md

```diff
@@ -20,7 +20,7 @@ or removed from the Kubernetes API.
 The policy has two settings:
 
 - `kubernetes_version`: The starting version from where to detect deprecated or
-  removed Kubernetes resources.
+  removed Kubernetes resources. This setting is mandatory.
```

Comment:
does this default to a particular value? If yes, that would be helpful to add.

Reply:
It doesn't. I suppose we need the user to decide on what version to test against.